core/cfg_test.yaml

```diff
@@ -0,0 +1 @@
+name: testCfgFile
```

Comment:
suggest to move files for test purpose to `./test/asset` folder (need to create one).